helmfile/values/service-cluster-networkpolicy.yaml.gotmpl

```
@@ -70,3 +71,9 @@ rcloneSync:
 
 s3Exporter:
   enabled: {{ and (eq .Values.objectStorage.type "s3") (and .Values.s3Exporter.enabled .Values.networkPolicies.s3Exporter.enabled) }}
+
+dex:
+  enabled: {{ .Values.networkPolicies.dex.enabled }}
+
+falco:
+  enabled: {{ .Values.networkPolicies.falco.enabled }}
```

```suggestion
falco:
  enabled: {{ .Values.networkPolicies.falco.enabled }}
```
And if falco itself is enabled.